modules/button/react/lib/ToolbarDropdownButton.tsx

```diff
@@ -0,0 +1,87 @@
+/** @jsx jsx */
+import {jsx} from '@emotion/core';
+import {colors, spacing, borderRadius} from '@workday/canvas-kit-react-core';
+import {focusRing, useTheme, Themeable, EmotionCanvasTheme} from '@workday/canvas-kit-react-common';
+import {SystemIcon} from '@workday/canvas-kit-react-icon';
+import {ButtonColors} from './types';
+import {ButtonContainer} from './parts';
+import {IconButtonProps} from './IconButton';
+import {chevronDownSmallIcon} from '@workday/canvas-system-icons-web';
+
+export interface ToolbarDropdownButtonProps
+  extends Omit<IconButtonProps, 'size' | 'variant' | 'toggled' | 'onToggleChange'>,
+    Themeable {}
+
+const ToolbarDropdownButton = ({
+  theme = useTheme(),
+  buttonRef,
+  'aria-label': iconArialabel,
+  icon,
+  children,
+  ...elemProps
+}: ToolbarDropdownButtonProps) => {
+  const containerStyles = {
+    padding: spacing.zero,
+    minWidth: spacing.l,
+    width: 'auto',
+    height: spacing.l,
+    borderRadius: borderRadius.m,
+    '& .wd-icon': {
+      display: 'inline-block',
+      verticalAlign: 'middle',
+      width: 20,
+      height: 20,
+    },
+  };
+
+  const chevronStyles = {
+    margin: '0 2px 0 0',
+  };
+
+  const customIconStyles = {
+    marginLeft: `${spacing.xxxs} !important`,
```

Comment:
ok so I added this cause button constainer adds
 ```
'& > *:first-of-type': {
      marginLeft: 0,
    },
```
I needed to override this per the design, thoughts on thios?

Reply:
I would just give this element a class below (e.g. `wdc-toolbar-dropdown-btn-arrow`) and move this to `containerStyles`. We should avoid `!important` wherever possible.